lightning/livekit_example/generate_audio.py

```diff
@@ -0,0 +1,37 @@
+import asyncio
+import aiohttp
+import wave
+from livekit.plugins import smallest
+
+from dotenv import load_dotenv
+load_dotenv()
+
+async def test_tts():
+    # Create an aiohttp session
+    async with aiohttp.ClientSession() as session:
+        # Create TTS instance with the session
+        tts = smallest.TTS(
```

Comment:
where is the smallest package mentioned ? ideally should be part of requirements

Reply:
part of livekit

```py
from livekit.plugins import smallest
```